### ISSUE TITLE
cobigen-cli shell wrappers fail to handle args with quote

### ISSUE THREAD

**Issue by markusschuh** [devonfw/tools-cobigen#1045]
### Expected behavior

As a user, I want to use cobigen-cli in a way so that even arguments containing quotes are handled correctly.

### Actual behavior

cobigen / cg separate arguments even with quoted spaces and such fail to identify files with space in path or filename.

### Steps to reproduce (bug) / Use Case of feature request (enhancement)

```
$ ./cg g "openapi file.xml"
[ERROR] Your <inputFile> 'openapi' has not been found
```

### Comments/Hints:

Use `"$@"` instead of `$*`. This forwards the correct quoting of arguments to other called programs. Hint: Use https://www.shellcheck.net/ to detect such issues.

Also you might replace the current check, if any argument is added, with a more often used code - even when the current check does not run into errors when quoted args are used.

Use either
```
if [ $# -eq 0 ] . . .
if [ -z "$1" ] . . .
```

Compare with #1044

**Comment by jdiazgon** [comment]
Thanks for the feedback and the hints to solve it! We will solve it ASAP.

**Comment by markusschuh** [comment]
You have missed, that it is already the `bash` wrapper scripts that fails to handle the quoted file correctly.  I have not yet checked the java code changes - this may workaround the error from the wrapper. But this won't generate stable results, I'd say.

Please fix it by changing
```
java -javaagent:$SOFTWARE_PATH/cobigen-cli/class-loader-agent.jar -jar $SOFTWARE_PATH/cobigen-cli/cobigen.jar $*
```

into

```
java -javaagent:$SOFTWARE_PATH/cobigen-cli/class-loader-agent.jar -jar $SOFTWARE_PATH/cobigen-cli/cobigen.jar "$@"
```

See https://github.com/koalaman/shellcheck/wiki/SC2048 for an example. Or check the bash man page.

**Comment by markusschuh** [comment]
Fixed by #1051. Thx
